namespaces/live-1.cloud-platform.service.justice.gov.uk/sentence-planning-development/resources/variables.tf

```diff
@@ -0,0 +1,35 @@
+variable "domain" {
+  default = "sentence-planning-development.service.justice.gov.uk"
```

Comment:
Same in here, you don't need multiple zones, can create one with "sentence-planning.service.justice.gov.uk"